benches/check.rs

```diff
@@ -24,5 +24,26 @@ pub fn check_benchmark(c: &mut Criterion) {
     });
 }
 
-criterion_group!(benches, check_benchmark);
+pub fn compare_benchmark(c: &mut Criterion) {
```

Comment:
Please move this benchmark function to `benches/compare.rs`.

Reply:
Will do